app/org/sagebionetworks/bridge/dynamodb/DynamoCriteriaDao.java

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+import javax.annotation.Resource;
+
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.dao.CriteriaDao;
+import org.sagebionetworks.bridge.models.Criteria;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+
+/**
+ * The DAO for managing this object, which is optionally associated with models that can be filtered by Criteria. Create
+ * and update are combined, and not finding an object does not throw an exception when retrieving or deleting.
+ */
+@Component
+public class DynamoCriteriaDao implements CriteriaDao {
+
+    private DynamoDBMapper criteriaMapper;
+    
+    @Resource(name = "criteriaMapper")
+    final void setCriteriaMapper(DynamoDBMapper criteriaMapper) {
+        this.criteriaMapper = criteriaMapper;
+    }
+    
+    // Until criteria as an interface is shifted off of SchedulePlan/Subpopulation, 
+    // we must copy from an interface to an implementation.
+    @Override
+    public Criteria copyCriteria(String key, Criteria criteria) {
```

Comment:
This has been moved to the criteria interface as a generic copy function and removed entirely from DAO.